Jenkinsfile

```diff
@@ -19,7 +19,7 @@ pipeline {
                 expression { env.CHANGE_ID != null } // Pull request
             }
             steps {
-                sh '${M2_HOME}/bin/mvn -B -V clean verify'
+                sh '${M2_HOME}/bin/mvn -B -V clean verify sonar:sonar -Prun-its'
```

Comment:
Sorry, forgot one stuff, need to add "-Dplugin.jacoco.skip=false" to enable jacoco for test coverage report. (Or add this property in your pom.xml)

Reply:
Sure, done.

Reply:
Now I can see the report on the boards: http://sonarqube-spmm-automation.apps.ocp4.prod.psi.redhat.com/projects